LFS171x/sawtooth-material/sawtooth-tuna/client/src/components.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
```

Comment:
Provide license information for this file.
`// SPDX-License-Identifier: Apache-2.0`